integration-tests/src/test/java/org/apache/fineract/integrationtests/common/GlobalConfigurationHelper.java

```diff
@@ -677,6 +692,12 @@ public static Integer updateIsAutomaticExternalIdGenerationEnabled(final Request
         return updateEnabledFlagForGlobalConfiguration(requestSpec, responseSpec, configId, enabled);
     }
 
+    public static Integer updateChargeAccrualDate(final RequestSpecification requestSpec, final ResponseSpecification responseSpec,
```

Comment:
Isn't this a dupe of updateChargeAccrualDateConfiguration() which is right below? That also tries to update the same configId 56.

Reply:
You are right! I dont know how i missed this :D